juniper_official/Solutions/EVPN-VXLAN/evpn-db-state-netconf.rule

```diff
@@ -0,0 +1,102 @@
+/*
+ * Rule 'verify-duplicate-mac' determine evpn database state has duplicate mac entry
+ * 
+ * One input control detection
```

Comment:
s/One input control detection/One input controls detection/g

Reply:
updated the rule